src/linux/py_proc.h

```diff
@@ -68,6 +68,21 @@ union {
 
 
 // ----------------------------------------------------------------------------
+static Elf64_Addr
+_py_proc_get_base64(Elf64_Ehdr * ehdr, void * elf_map)
+{
+  Elf64_Addr res = (Elf64_Addr) 0 - 1;
+  for (uint16_t i = 0; i < ehdr->e_phnum; ++i) {
```

Comment:
Could you please use a `register int` for the loop.

Reply:
Isn't `register` useless for modern compilers?

Reply:
It probably is. I'll leave it up to you. I would still use a plain `int` though, as the variable is only allocated in the stack and will take up the same space anyway because of alignment.